airflow/ui/src/pages/Dashboard/Stats/DAGImportErrorsModal.tsx

```diff
@@ -0,0 +1,131 @@
+};
+
+const PAGE_LIMIT = 5;
+
+export const DAGImportErrorsModal: React.FC<ImportDAGErrorModalProps> = ({
+  onClose,
+  onErrorCountChange,
+  open,
+}) => {
+  const [currentPage, setCurrentPage] = useState(0);
+
+  const { data } = useImportErrors({
+    limit: PAGE_LIMIT,
+    offset: currentPage * PAGE_LIMIT,
+  });
+
+  const importErrors: Array<ImportErrorResponse> = data.import_errors;
+  const importErrorsCount: number = data.total_entries || 0;
+  const totalPages = Math.ceil(importErrorsCount / PAGE_LIMIT);
+
+  useEffect(() => {
+    onErrorCountChange(importErrorsCount);
+  }, [importErrorsCount, onErrorCountChange]);
+
+  useEffect(() => {
+    if (!open) {
+      setCurrentPage(0);
+    }
+  }, [open]);
+
+  const handleNextPage = () => {
+    if (currentPage < totalPages - 1) {
+      setCurrentPage((prev) => prev + 1);
+    }
+  };
+
+  const handlePreviousPage = () => {
+    if (currentPage > 0) {
+      setCurrentPage((prev) => prev - 1);
+    }
+  };
+
+  return (
+    <Dialog.Root onOpenChange={onClose} open={open} size="xl">
+      <Dialog.Content backdrop>
+        <Dialog.Header>
+          <VStack align="start" gap={4} position="sticky" top={0} zIndex={1}>
+            <Heading size="xl">DAG Import Errors</Heading>
+            <Text color="gray.500">{`Page ${currentPage + 1} of ${totalPages}`}</Text>
+          </VStack>
+        </Dialog.Header>
+
+        <Dialog.CloseTrigger />
+
+        <Dialog.Body maxH="500px" overflowY="auto">
+          <Accordion.Root collapsible multiple size="md" variant="enclosed">
+            {importErrors.map((importError: ImportErrorResponse) => (
+              <Accordion.Item
+                key={importError.import_error_id}
+                value={importError.filename}
+              >
+                <Accordion.ItemTrigger cursor="pointer">
+                  <PiFilePy />
+                  {importError.filename}
+                </Accordion.ItemTrigger>
+                <Accordion.ItemContent>
+                  <Text color="gray.500" fontSize="sm" mb={1}>
+                    Timestamp : <Time datetime={importError.timestamp} />
+                  </Text>
+                  <Text color="red.600" fontSize="sm" whiteSpace="pre-wrap">
+                    <code>{importError.stack_trace}</code>
+                  </Text>
+                </Accordion.ItemContent>
+              </Accordion.Item>
+            ))}
+          </Accordion.Root>
+        </Dialog.Body>
+
+        <Box p={6}>
+          <HStack justify="space-between">
+            <Button
+              colorPalette="blue"
+              disabled={currentPage === 0}
+              onClick={handlePreviousPage}
+            >
+              Previous
+            </Button>
+            <Button
+              colorPalette="blue"
+              disabled={currentPage === totalPages - 1}
+              onClick={handleNextPage}
+            >
+              Next
+            </Button>
+          </HStack>
+        </Box>
```

Comment:
We already have a `Pagination` component you can use. [Chakra docs](https://www.chakra-ui.com/docs/components/pagination)